Date:   Mon Jan 15 22:24:52 2024 +0900

    add ttyd-sockjs.js and the code to use the file.

diff --git a/src/components/terminal/index.tsx b/src/components/terminal/index.tsx
--- a/src/components/terminal/index.tsx
+++ b/src/components/terminal/index.tsx
@@ -12,6 +12,8 @@ import { ZmodemAddon, FlowControl } from '../zmodem';
 import 'xterm/css/xterm.css';
 import worker from 'xterm-addon-image/lib/xterm-addon-image-worker';
 
+const { TtydSockJS: WebSocket } = require('./ttyd-sockjs');
+
 const imageWorkerUrl = window.URL.createObjectURL(new Blob([worker], { type: 'text/javascript' }));
 
 interface TtydTerminal extends Terminal {
